tox error: no module named 'hyperopt'

```
$ tox
GLOB sdist-make: /home/jeff/git/scVI/setup.py
py36 inst-nodeps: /home/jeff/git/scVI/.tox/dist/scvi-0.3.0.zip
py36 installed: anndata==0.6.19,atomicwrites==1.3.0,attrs==19.1.0,backcall==0.1.0,bleach==3.1.0,cycler==0.10.0,decorator==4.4.0,defusedxml==0.5.0,entrypoints==0.3,h5py==2.9.0,ipykernel==5.1.0,ipython==7.4.0,ipython-genutils==0.2.0,ipywidgets==7.4.2,jedi==0.13.3,Jinja2==2.10.1,jsonschema==3.0.1,jupyter==1.0.0,jupyter-client==5.2.4,jupyter-console==6.0.0,jupyter-core==4.4.0,kiwisolver==1.0.1,llvmlite==0.28.0,loompy==2.0.17,MarkupSafe==1.1.1,matplotlib==3.0.3,mistune==0.8.4,more-itertools==7.0.0,natsort==6.0.0,nbconvert==5.4.1,nbformat==4.4.0,notebook==5.7.8,numba==0.43.1,numpy==1.14.6,pandas==0.24.2,pandocfilters==1.4.2,parso==0.4.0,pexpect==4.7.0,pickleshare==0.7.5,pkg-resources==0.0.0,pluggy==0.9.0,prometheus-client==0.6.0,prompt-toolkit==2.0.9,ptyprocess==0.6.0,py==1.8.0,Pygments==2.3.1,pyparsing==2.4.0,pyrsistent==0.14.11,pytest==4.4.1,python-dateutil==2.8.0,pytz==2019.1,pyzmq==18.0.1,qtconsole==4.4.3,scikit-learn==0.19.2,scipy==1.2.1,scvi==0.3.0,seaborn==0.9.0,Send2Trash==1.5.0,six==1.12.0,terminado==0.8.2,testpath==0.4.2,torch==1.0.1.post2,tornado==6.0.2,tqdm==4.31.1,traitlets==4.3.2,umap-learn==0.3.8,wcwidth==0.1.7,webencodings==0.5.1,widgetsnbextension==3.4.2,xlrd==1.2.0
py36 runtests: PYTHONHASHSEED='1772156136'
py36 runtests: commands[0] | pytest
====================================================== test session starts =======================================================
platform linux -- Python 3.6.7, pytest-4.4.1, py-1.8.0, pluggy-0.9.0
rootdir: /home/jeff/git/scVI, inifile: setup.cfg
collected 10 items / 2 errors / 8 selected                                                                                       

============================================================= ERRORS =============================================================
______________________________________________ ERROR collecting tests/test_scvi.py _______________________________________________
ImportError while importing test module '/home/jeff/git/scVI/tests/test_scvi.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_scvi.py:9: in <module>
    from scvi.benchmark import all_benchmarks, benchmark, benchmark_fish_scrna, ldvae_benchmark
scvi/benchmark.py:5: in <module>
    from scvi.inference import UnsupervisedTrainer, TrainerFish
scvi/inference/__init__.py:14: in <module>
    from .autotune import auto_tune_scvi_model, hyperopt_worker, launch_workers
scvi/inference/autotune.py:18: in <module>
    from hyperopt import fmin, tpe, Trials, hp, STATUS_OK
E   ModuleNotFoundError: No module named 'hyperopt'
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,6 @@ language: python
 
 python:
   - 3.7
-  - 3.6
 
 os:
   - linux

--- a/CONTRIBUTING.rst
+++ b/CONTRIBUTING.rst
@@ -112,7 +112,7 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 3.6. Check
+3. The pull request should work for Python 3.7. Check
    https://travis-ci.org/YosefLab/scVI/pull_requests
    and make sure that the tests pass for all supported Python versions.
 

--- a/README.rst
+++ b/README.rst
@@ -26,7 +26,7 @@ Single-cell Variational Inference
 Quick Start
 -----------
 
-1. Install Python 3.6 or Python 3.7. We typically use the Miniconda_ Python distribution.
+1. Install Python 3.7. We typically use the Miniconda_ Python distribution.
 
 .. _Miniconda: https://conda.io/miniconda.html
 

--- a/readthedocs.yml
+++ b/readthedocs.yml
@@ -4,5 +4,5 @@ build:
   image: latest
 
 python:
-  version: 3.6
+  version: 3.7
   setup_py_install: true

--- a/setup.py
+++ b/setup.py
@@ -48,7 +48,7 @@ setup(
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
-        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Microsoft :: Windows',
         'Operating System :: POSIX :: Linux',

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist=py36,py37,travis
+envlist=py37,travis
 
 [testenv]
 deps=pytest
@@ -7,7 +7,7 @@ extras = test
 commands=pytest
 
 [testenv:travis]
-basepython=python3.6
+basepython=python3.7
 deps = -rrequirements.txt
 extras =
 commands =  flake8 scvi/ tests/ ./*.py

